### ISSUE TITLE
Softmax before nn.CrossEntropyLoss()

### ISSUE THREAD

**Issue by bhacquin** [sugi-chan/custom_bert_pipeline#1]
I'm just wondering about the use of the function Softmax before applying the CrossEntropyLoss. I reckon the loss has a softmax built-in. So I am not quite sure what is the effect of adding another softmax before (Cell 21 in the definition of train_model).
I
